<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeybindSettings</class>
 <widget class="QDialog" name="KeybindSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>190</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>190</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>190</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Recording Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #202020;</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #303030;
color: #FFFFFF;
border: 1px solid #008080;
border-radius: 0px;
width: 75%;
height:25%;
</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="recordInput">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {border: 1px solid #008080;}

QCheckBox::indicator:checked {background: #009090;}

QCheckBox{color: #FFFFFF;}</string>
   </property>
   <property name="text">
    <string>Record Input Audio</string>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="recordLoopback">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {border: 1px solid #008080;}

QCheckBox::indicator:checked {background: #009090;}

QCheckBox{color: #FFFFFF;}</string>
   </property>
   <property name="text">
    <string>Record Loopback</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="padAudio">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {border: 1px solid #008080;}

QCheckBox::indicator:checked {background: #009090;}

QCheckBox::indicator:disabled{background: #606060;}

QCheckBox::disabled{color:#606060;}

QCheckBox{color: #FFFFFF;}</string>
   </property>
   <property name="text">
    <string>Pad Loopback</string>
   </property>
  </widget>
  <widget class="QToolButton" name="browse">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #008080;
background: #303030;
color: #FFFFFF;</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="url">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #008080;
background: #303030;
color: #FFFFFF;</string>
   </property>
   <property name="text">
    <string>Select a sample...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>KeybindSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>KeybindSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
